fixtures/dom/src/components/Fixture.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
For `React` and `ReactDOM` they should just be globally available. If we import it then we're only getting the current version that `create-react-app` installed. The `react-loader` script will [load the selected version](https://github.com/facebook/react/blob/master/fixtures/dom/public/react-loader.js#L31-L32) from a CDN.